readme.md

```diff
@@ -234,9 +234,80 @@ Ink uses [Yoga](https://github.com/facebook/yoga) - a Flexbox layout engine to b
 It's important to remember that each element is a Flexbox container.
 Think of it as if each `<div>` in the browser had `display: flex`.
 See `<Box>` built-in component below for documentation on how to use Flexbox layouts in Ink.
+Note that all text must be wrapped in `<Text>` component.
 
 ### Built-in Components
 
+#### `<Text>`
+
+This component can display text, and change its style to make it bold, underline, italic or strikethrough.
+
+```jsx
+import {Text} from 'ink';
+
+const Example = () => (
+	<>
+		<Text bold>I am bold</Text>
+		<Text italic>I am italic</Text>
+		<Text underline>I am underline</Text>
+		<Text strikethrough>I am strikethrough</Text>
+	</>
+);
+```
+
+##### bold
+
+Type: `boolean`<br>
```

Comment:
Unrelated to this PR, but just FYI, you can now use `\` instead of `<br>` to force a linebreak.
```suggestion
Type: `boolean`\
```